### DISCUSSION TITLE
$inertia.form helper should have access to initial value

### DISCUSSION THREAD

**Discussion by khanalpride** [inertiajs/inertia#376] [Ideas]
I guess $inertia.form should have initial values to be accessible so that it can be used to check if field is dirty,

Like jetstream form helper
https://github.com/laravel/jetstream-js/blob/master/src/InertiaForm.js#L115

**Comment by reinink** [comment]
I've started a new PR for this feature: https://github.com/inertiajs/inertia/pull/660
